frontend/src/scenes/feature-flags/FeatureFlags.tsx

```diff
@@ -219,6 +219,33 @@ function OverViewTab(): JSX.Element {
                         value={searchTerm}
                     />
                     <div className="flex items-center gap-2">
+                        {!hasAvailableFeature(AvailableFeature.MULTIVARIATE_FLAGS) && (
+                            <>
+                                <span>
+                                    <b>Type</b>
+                                </span>
+                                <LemonSelect
+                                    onChange={(type) => {
+                                        if (type) {
+                                            if (type === 'all') {
+                                                if (filters) {
+                                                    const { type, ...restFilters } = filters
+                                                    setFeatureFlagsFilters(restFilters, true)
+                                                }
+                                            } else {
+                                                setFeatureFlagsFilters({ type })
+                                            }
+                                        }
+                                    }}
+                                    options={[
+                                        { label: 'All', value: 'all' },
+                                        { label: 'Multiple variants', value: 'multivariant' },
```

Comment:
why the discrimination against boolean/single variant flags? :P

(as in, why not have them as a filter option too?)